### DISCUSSION TITLE
removing web3auth logo.

### DISCUSSION THREAD

**Discussion by irfanonk** [Web3Auth/Web3Auth#1155] [Help]
Hi. 
Is it possible to remove logo and customize web3Auth popup in Plug and Play  PAID version?

**Comment by Saviour1001** [comment]
Yes, depending upon the SDK you are using, you can find the [Whitelabel section](https://web3auth.io/docs/sdk/web/modal/whitelabel) in the docs which has code snippets and implementations to customize.
